ports/esp32/sdkconfig.h

```diff
@@ -23,7 +23,7 @@
 #define CONFIG_ESP32_DEFAULT_CPU_FREQ_MHZ 240
 #define CONFIG_ESP32_DEFAULT_CPU_FREQ_240 1
 #define CONFIG_ESP32_DEBUG_OCDAWARE 1
-#define CONFIG_ESP32_DEEP_SLEEP_WAKEUP_DELAY 0
+#define CONFIG_ESP32_DEEP_SLEEP_WAKEUP_DELAY 2000
```

Comment:
What exactly does this do?  What is 2000 measured in, seconds?

Reply:
> What exactly does this do?

Discussion here: https://github.com/micropython/micropython-esp32/issues/32#issuecomment-333501105

2000 is now the default value (instead of 0) in the ESP-IDF as of 9 months ago: https://github.com/espressif/esp-idf/commit/3c583a76553856091305ffec39194711efb4cb10

Reply:
Ok, thanks, understood.